docs/patterns/components/Accordion/index.css

```diff
@@ -0,0 +1,11 @@
+.Accordion__container {
+  max-width: 600px;
+}
+
+.Accordion__panel {
+  min-height: 200px;
```

Comment:
I forgot, do we usually convert px to rem?

Reply:
I can definitely do that - this is only included for the demo purposes though, just as an FYI

Reply:
Ah gotcha. Not a big deal.